assets/package.json

```diff
@@ -6,7 +6,9 @@
   "main": "./assets/js/phoenix_live_dashboard.js",
   "repository": {},
   "scripts": {
-    "build": "webpack --mode production",
+    "build": "npm run build.production && npm run build.development",
+    "build.production": "webpack --mode production --env.NODE_ENV=production",
+    "build.development": "webpack --mode production --env.NODE_ENV=development",
```

Comment:
Instead of using NODE_ENV, could we customize the mode instead and use --mode production and --mode development? Both emit optimized code, but one with source map and another without.

Reply:
I'll check that but it may be to many defaults to override

Reply:
Ok, nevermind. :)

Reply:
Maybe what we should do is to rename the environment variable. Instead of NODE_ENV, use something like `NODE_SOURCE_MAPS`. That may be clearer.

Reply:
And then we simply check the source maps option, regardless of the mode.